lib/SmartyPlugins/modifier.duration_format.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * LMS version 1.11-git
+ *
+ *  (C) Copyright 2001-2021 LMS Developers
+ *
+ *  Please, see the doc/AUTHORS for more information about authors!
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License Version 2 as
+ *  published by the Free Software Foundation.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307,
+ *  USA.
+ *
+ *  $Id$
+ */
+
+function smarty_modifier_duration_format($sec)
+{
+    $H = floor($sec / 3600);
+    $m = ($sec / 60) % 60;
+    $s = $sec % 60;
+    if ($sec < 60) {
+        return sprintf("%02ds", $s);
+    } elseif ($sec < 3600) {
+        return sprintf("%02dm %02ds", $m, $s);
+    } else {
+        return sprintf("%dh %02dm %02ds", $H, $m, $s);
```

Comment:
Sztywno zakładasz, że ma być notacja `2h 43m 28s`, a co jak w innym kraju nie zapisuje się tego w ten sposób?
Może warto w Localisation.php przygotować ustawienia tego co bierzemy w sprintf pod każdy kraj indywidualnie.

Reply:
Czy czasami notacja 1g zamiast 1h nie byłaby bardziej przyjazna?

Ja nie wiem ile osób z poza PL używa LMSa...myślę, że Ty to jesteś w stanie bardziej określić. Poza tym to jest błachostka. Inne rzeczy bardziej przeszkadzają.

Reply:
Nie mam koncepcji "na już" jak to zrobić. Może uradzisz commit'em?